content/help/security-vulnerabilities/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio Security Vulnerability Disclosure Information
+description: Responsible disclosure for Istio security vulnerabilities
+weight: 35
+aliases:
+    - /security-vulnerabilities.html
+    - /security-vulnerabilities/index.html
+---
+## How to report a Security Vulnerability
```

Comment:
I think the convention is to only capitalize the first word (i.e., not for 'Security Vulnerability'). Same below.

Reply:
+1

Reply:
Done: the capitalization of the words have been fixed.